userconfigs_generator/generator_test_expected.txt

```diff
@@ -0,0 +1,484 @@
+// Code generated by user config generator. DO NOT EDIT.
```

Comment:
since this isn't really a text file, could the file be renamed to have `.go` extension in it? you could also consider something like `.tmp.go`, or `.unused.go`, etc.